index.html

```diff
@@ -145,28 +145,37 @@ <h2>
                   </p>
                   <!-- <h3 class="lets-connect">LET'S CONNECT</h3> -->
                   <ul>
-                    <a href="https://twitter.com/SolomonAfutu"
+                    <li>
+                      <a href="https://twitter.com/Dakingzman"
                       ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
                     ></a>
-                    <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+                    </li>
+                    <li>
+                      <a href="https://www.linkedin.com/in/kingsley-chidozie-igbor/"
                       ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
                     ></a>
-                    <a href="https://github.com/SolomonTheCoder"
+                    </li>
+                    <li>
+                      <a href="https://github.com/Developer-kingz"
                       ><i class="fa fa-github fa-2x" aria-hidden="true"></i
                     ></a>
-                    <a href="">
-                      <i class="fa fa-medium fa-2x" aria-hidden="true"></i
-                    ></a>
-                    <a href=""
+                    </li>
+                    <li>
+                      <a href="">
+                        <i class="fa fa-medium fa-2x" aria-hidden="true"></i
+                      ></a>
+                    </li>
+                    <li>
+                      <a href=""
                       ><i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i
                     ></a>
+                    </li>
                   </ul>
                   <button type="button" class="see-btn">Get my resume</button>
                 </div>
                 <div class="Language">
-                  <div class="lang flex-row">
-                    <h2>Languages</h2>
-                    <i class="fa fa-angle-down fa-2x" aria-hidden="true"></i>
+                  <div class="lang">
+                    <p class="languages">Languages</p>
```

Comment:
- [ ] hey you have done a great job on this project pleas kindly take more look to the original figma template its good to stick to the design according to the reqiurement ,
- [ ] According to the previous reviewer Please adhere to the project requirement and do also follow the Figma design. As your previous code reviewer suggested you to made some changes. You did some of them but still have not implemented the changes in your language section. It is very important to make your design looks similar to the given design on Figma because it will help you to be a better developer who is also able to implement the project detail in a professional way. Kindly take a look at the below table in order to see the difference 🔍
- [ ] your design
![hhhhh](https://user-images.githubusercontent.com/92458236/236352328-e8fc068c-2dce-4285-8a14-9fae95ab9928.jpg)
 - [ ] figma design
![kllllll](https://user-images.githubusercontent.com/92458236/236352367-4bc88162-04f7-4ac8-bc84-3a5f4d93d6f3.jpg)